components/camel-grpc/src/main/java/org/apache/camel/component/grpc/GrpcConstants.java

```diff
@@ -50,4 +50,9 @@
     String GRPC_EVENT_TYPE_ON_NEXT = "onNext";
     String GRPC_EVENT_TYPE_ON_ERROR = "onError";
     String GRPC_EVENT_TYPE_ON_COMPLETED = "onCompleted";
+
+    /*
+     * The registry key to lookup a custom BindableServiceFactory
+     */
+    String GRPC_BINDABLE_SERVICE_FACTORY_NAME = "bindableServiceFactory";
```

Comment:
I wonder if we should prefix it with grpc, eg grpcBindableServiceFactory

Reply:
Yeah, good point. I'll do that.